### ISSUE TITLE
Black screen

### ISSUE THREAD

**Issue by devilgugu2929** [tuffee88/d3d12ProxyEdrDx11_0#36]
So I applied the fix and it got me past the white screen but now there is a black screen with music. I managed to run it by using dxcpl as mentioned in the other post and by disabling my GPU but the fps was so bad.I have a gt 730 ddr3 Fermi version. My driver version is 391.95 because of Fermi card.Is that the issue.Iam willing to do anything to make it run. Thanks.

**Comment by tuffee88** [comment]
This is either a driver or - more likely- just plain missing hardware functionality on Fermi cards.

If you *really* want to investigate why this is happening I'd suggest to check:
a) If this is a "shading issue" of sorts - you can just use ReShade and select a shader that makes the dept information or wireframe view visible -> If this is indeed shading related and your GPU does render the scene geometry at least you should see something that resembles Elden Ring.
b) If a) works get a GPU capture using Microsoft Pix: Try to play it back on your GPU with validation and check the error messages.  Depending on what exactly is happening you might be able to verify that the problems are indeed fixable by modifying the shader code in question in Pix. 

*If* the changes are straightforward I *might* be able to inject some code in the game rendering to replicate the fix and make this render somewhat properly but that's a big if - and to figure this out you'd need to have/acquire some GPU rendering debug experience.

**Comment by devilgugu2929** [comment]
Which shader should I select?

**Comment by devilgugu2929** [comment]
Which shader should I select?

**Comment by tuffee88** [comment]
Try "DisplayDepth" (part of the standard shaders). This should give you a split image with a sort of wireframe view on the left and depth/distance shading on the right.

**Comment by devilgugu2929** [comment]
Reshade works fine with other games but It is not working for elden ring. The menu does not open.

**Comment by tuffee88** [comment]
ReShade works as expected for me. 
Are you using the newest release of this project and the windows/system32 d3d12.dll (as mentioned in the README) ?

**Comment by devilgugu2929** [comment]
Hello I discovered something. I was just browsing the driver folder and I find some vulkan drivers. After installing them the game still didn't show the display but after some time it said this
Vulkan_dll error.The file is corrupted or not meant for this use.
Does this have to do something?

**Comment by devilgugu2929** [comment]
> ReShade works as expected for me. Are you using the newest release of this project and the windows/system32 d3d12.dll (as mentioned in the README) ?

Yes the game just gives me 0x0000b(don't know if I typed that correctly) error if I use the SysWow64 one

**Comment by tuffee88** [comment]
Ah, I see - Please try this without the d3d12_original.dll inside the game folder 

 -> ReShade hooks calls to d3d12 inside of windows/system32, if we use a copy of the very same dll inside the game folder ReShade won't be able to load correctly.

**Comment by devilgugu2929** [comment]
I noticed that once you open reshade menu whole screen starts flickering

**Comment by tuffee88** [comment]
Thanks for testing. Tbh this doesn't look good - it's not a simple shading problem, as you usually see geometry rendered then (dept buffer is populated any you can see a sort of wireframe view). I won't be able to properly reproduce this (no Fermi gpu), so I sadly for now it seems you're out of luck :-(

**Comment by devilgugu2929** [comment]
> Thanks for testing. Tbh this doesn't look good - it's not a simple shading problem, as you usually see geometry rendered then (dept buffer is populated any you can see a sort of wireframe view). I won't be able to properly reproduce this (no Fermi gpu), so I sadly for now it seems you're out of luck :-(

Thanks for helping.Will I be able to play it in future in case fromsoft launches a dx11 API or something?

**Comment by tuffee88** [comment]
> > Thanks for testing. Tbh this doesn't look good - it's not a simple shading problem, as you usually see geometry rendered then (dept buffer is populated any you can see a sort of wireframe view). I won't be able to properly reproduce this (no Fermi gpu), so I sadly for now it seems you're out of luck :-(
> 
> Thanks for helping.Will I be able to play it in future in case fromsoft launches a dx11 API or something?

If the introduce a DX12 path with lower hardware requirements: Yes, but I wouldn't count on it. 
It's more likely that someone with a Fermi card takes the time & effort to mod the game to support that, though (just my 2cts).

**Comment by devilgugu2929** [comment]
I have seen some other people play on Fermi cards
https://youtu.be/dzz1PNFFDac
https://youtu.be/abMjkczBAEM

**Comment by tuffee88** [comment]
> I have seen some other people play on Fermi cards https://youtu.be/dzz1PNFFDac https://youtu.be/abMjkczBAEM
If you're talking about NVIDIA GT 730s: There are at least 3 different chips used for them with bigger real-world feature and performance differences, two of them use Kepler and one is still Fermi (according to Wikipedia).

**Comment by WesleyBrandon** [comment]
Hi Tuffee88, have you managed to find out what it could be? I have exactly the same GPU and the same problem as this person.
